modules/hivemq-edge-module-opcua/src/main/java/com/hivemq/edge/adapters/opcua/OpcUaAdapterConfig.java

```diff
@@ -34,6 +34,13 @@ public class OpcUaAdapterConfig extends AbstractProtocolAdapterConfig {
                        required = true)
     private @NotNull String uri;
 
+    @JsonProperty("updateEndpoint")
+    @ModuleConfigField(title = "Update Endpoint",
+                       description = "Updates endpoint with hostname and port from given URI.",
+                       format = ModuleConfigField.FieldType.BOOLEAN,
+                       required = false)
+    private @NotNull Boolean updateEndpoint = true;    
+
```

Comment:
I'd like to rename this, to make it more clear what the intention is.

```suggestion
    @JsonProperty("overrideUri")
    @ModuleConfigField(title = "Override server returned endpoint URI",
                       description = "Overrides the endpoint URI returned from the OPC-UA server with the hostname and port from the specified URI.",
                       format = ModuleConfigField.FieldType.BOOLEAN,
                       required = false)
    private @NotNull Boolean overrideUri = true;    

```

Reply:
added it to my branch